modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/maintenance/ClearFolderWorkflow.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2023 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache.persistence.maintenance;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.ignite.maintenance.MaintenanceAction;
+import org.apache.ignite.maintenance.MaintenanceRegistry;
+import org.apache.ignite.maintenance.MaintenanceWorkflowCallback;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * The workflow is used for {@link ClearFolderAction} in maintenance mode.
+ */
+public class ClearFolderWorkflow implements MaintenanceWorkflowCallback {
+    /** Task name. */
+    public static final String CLEAR_FOLDER_TASK = "cleanForderTask";
```

Comment:
A minor typo: "Forder". I also suggest to unify wording with `ClearFolderAction` and use the word "clear" instead of "clean" used now.